src/main.py

```diff
@@ -16,17 +16,32 @@
 from selenium.webdriver.chrome.service import Service
 
 # Load in the credentials from .env
-load_dotenv()
+load_dotenv(find_dotenv())
 
 # Set the email and password variables from the environment variables
 email = os.getenv('EMAIL')
 password = os.getenv('PASSWORD')
 token = os.getenv('TOKEN')
 newpassword = os.getenv('NEWPASSWORD')
 
-def totp(gen):
+def main(mode,code):

```

Comment:
It would be better to split the individual "modes" into their own functions, e.g `def totp`, and `def pr` (Note: add comments which explain what the mode abbreviation's full forms are)

E.g:

```py
def totp(code):
	#totp logic

def pr(code):
	#password reset logic

def main():
	umode = input("Enter the program mode")
	if umode == "totp":
		totp(code)
	elif umode == "pr":
		pr(code)
```

This would make each individual feature and mode easier to maintain and debug.